internal/magic/text.go

```diff
@@ -251,6 +251,32 @@ func NdJson(raw []byte, limit uint32) bool {
 	return parsed > 2 && parsed == len(raw)
 }
 
+// Har matches a HAR Spec file.
+// Spec: http://www.softwareishard.com/blog/har-12-spec/
+func HAR(raw []byte, limit uint32) bool {
```

Comment:
I think it's worth checking the first element inside `log` is one of `version`, `creator`, `browser`, `pages`, `entries`, `comment`. It would help with false positive cases where a simple JSON file just happens to contain `log` as it's first key.

Reply:
It has been changed to checking the first element inside `log` contains one of the non-optional elements `version`, `creator`, and `entries`.